main/dsp.c

```diff
@@ -1223,7 +1223,7 @@ int ast_dsp_call_progress(struct ast_dsp *dsp, struct ast_frame *inf)
 		return 0;
 	}
 	if (!ast_format_cache_is_slinear(inf->subclass.format)) {
-		ast_log(LOG_WARNING, "Can only check call progress in signed-linear frames\n");
+		ast_log(LOG_WARNING, "Can only check call progress in signed-linear frames, %s not supported\n", ast_format_get_name(inf->subclass.format));
```

Comment:
These lines are pretty long now. I might look nicer to add a line break after the format string. This applies to all of the logging statement changes you made in this file.